nrcemt/qeels/gui/main_window.py

```diff
@@ -287,6 +318,7 @@ def save_results(self):
             save_results(save_path, headers, data)
 
     def detect(self):
+        ev_vals = (15, 15.8, 16.7, 24.8, 25, 33)
```

Comment:
this along `material_options` should probs be constants in all caps at the top of the file